downstream/modules/builder/con-definition-dependencies.adoc

```diff
@@ -1,6 +1,7 @@
 [id="con-definition-dependencies"]
 
 = Dependencies
+To avoid issues with your {ExecEnvNameSing} image, make sure your definition file adheres to all Galaxy, Python and system dependency requirements.
```

Comment:
```suggestion
To avoid issues with your {ExecEnvNameSing} image, make sure your definition file adheres to all Galaxy, Python, and system dependency requirements.
```

Reply:
Do we want to mention "Automation Hub" instead of Galaxy?

Reply:
I'm not sure whether to mention Hub versus Galaxy? Other changes complete.